src/Aspire.Dashboard/Components/Pages/Metrics.razor

```diff
@@ -12,9 +12,11 @@
 
 <PageTitle><ApplicationName ResourceName="@nameof(Dashboard.Resources.Metrics.MetricsPageTitle)" Loc="@Loc" /></PageTitle>
 
-<div class="metrics-layout">
-    <h1 class="page-header">@Loc[nameof(Dashboard.Resources.Metrics.MetricsHeader)]</h1>
-    <FluentToolbar Orientation="Orientation.Horizontal">
+<AspirePageContentLayout AddNewlineOnDesktopToolbar="true">
```

Comment:
The meter/instrument area scrollbar isn't displayed. Instead, the whole page because scrollable.

![image](https://github.com/dotnet/aspire/assets/303201/8b155692-f8c8-46d5-a0be-0210aba24cbd)

Reply:
Also, the horizontal scrollbar for the graph area (it required because of the browser size) isn't visible unless you scroll the page to the bottom.

Reply:
fixed